scripts/whitelist-file.ts

```diff
@@ -0,0 +1,120 @@
+// Simple upload 500 addresses to whitelist using whitelist_addresses.csv.
+// Accepts cosmos, stars addresses.
+// If you run into an error with `member_limit`, run `yarn whitelist --increase-member-limit`
+
+import { ExecuteMsg } from '@stargazezone/types/contracts/whitelist/execute_msg';
+import {
+  coins,
+  DirectSecp256k1HdWallet,
+  GasPrice,
+  MsgExecuteContractEncodeObject,
+  SigningCosmWasmClient,
+} from 'cosmwasm';
+import { toStars } from '../src/utils';
+import inquirer from 'inquirer';
+import { getClient } from '../src/client';
+import { MsgExecuteContract } from 'cosmjs-types/cosmwasm/wasm/v1/tx';
+import { toUtf8 } from '@cosmjs/encoding';
+import * as fs from 'fs';
+import * as path from 'path';
+import { parse } from 'csv-parse';
+
+const config = require('../config');
+const WHITELIST_CREATION_FEE = coins('100000000', 'ustars');
+const MSG_ADD_ADDR_LIMIT = 500;
+
+const gasPrice = GasPrice.fromString('0ustars');
+
+async function addFile() {
+  const wallet = await DirectSecp256k1HdWallet.fromMnemonic(config.mnemonic, {
+    prefix: 'stars',
+  });
+
+  type Whitelist = {
+    address: string;
+  };
```

Comment:
```suggestion
  interface Whitelist {
    address: string;
  }
```

This is the recommended way to create types for objects